Automattic/jetpack
# Milestone widget: avoid php notices

Fixes #20208

#### Changes proposed in this Pull Request:

Although the root of the issue may be addressed in Core in the next few days (see [this Trac ticket](https://core.trac.wordpress.org/ticket/53548)), this PR puts us on the safe side by adding extra checks to avoid PHP notices that may be created by the Milestone widget.

One of the notices was mentioned in #20208, but there were also others:

```
PHP Notice:  Undefined index: title in modules/widgets/milestone/class-milestone-widget.php on line 220
PHP Notice:  Undefined index: event in modules/widgets/milestone/class-milestone-widget.php on line 245
```

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start from a site using WP 5.8 Beta.
* Go to Jetpack > Settings and ensure the extra sidebar widgets feature is active.
* Go to Appearance > Widgets and add a new milestone widget to your sidebar.
* Ensure it can be configured and displayed on the frontend.
* Ensure no notices are logged.
